crates/speedy-transform/src/web_transform/visit.rs

```diff
@@ -35,11 +37,5 @@ impl Visit for IdentComponent {
       .push((ident.sym.to_string(), ident.span.ctxt.as_u32()));
   }
 
-  fn visit_ts_type_ref(&mut self, ts_type: &TsTypeRef) {
-    if let TsEntityName::Ident(ident) = &ts_type.type_name {
-      self
-        .ident_list
-        .push((ident.sym.to_string(), ident.span.ctxt.as_u32()));
-    }
-  }
+  fn visit_ts_type_ref(&mut self, _: &TsTypeRef) {}
```

Comment:
You can use `swc_ecma_visit::noop_visit_type` to skip all type visits which speeds up visiting